stories/navbar/navbar.stories.mdx

```diff
@@ -264,6 +267,128 @@ You can also choose to render a custom component of your choice as an action but
     </Story>
 </Canvas>
 
+<Heading3>Custom component items</Heading3>
+
+You can also choose to render a custom component of your choice as an items.
+
+<Canvas>
+    <Story name="Custom items">
+        {() => {
+            const navbarRef = useRef();
+            const [selected, setSelected] = useState("home");
+            return (
+                <Navbar
+                    ref={navbarRef}
+                    items={{
+                        desktop: [
```

Comment:
can you also provide an example of a custom component on desktop? doesn't have to be as complicated as the mobile one, a simple icon will do